src/lib/tvgShapeImpl.h

```diff
@@ -213,6 +217,9 @@ struct Shape::Impl
 
     bool dispose(RenderMethod& renderer)
     {
+        if (comp && eCompData) {
```

Comment:
Paint class who has comp, must handle this, not here.

Reply:
moved